app/javascript/components/batch_update/components/CurrentCriteria.js

```diff
@@ -111,6 +124,27 @@ function CurrentKeywords(props) {
   )
 }
 
+function CurrentPrice(props) {
+  const { minPrice, onRemoveMinPrice, maxPrice, onRemoveMaxPrice } = props
+  return (
+    <div>
+      <h2>Price</h2>
+      {minPrice && (
+        <SelectedPrice
+          name={'Min Price: ' + minPrice}
+          onRemove={onRemoveMinPrice}
+        />
+      )}
+      {maxPrice && (
+        <SelectedPrice
+          name={'Min Price: ' + maxPrice}
+          onRemove={onRemoveMaxPrice}
+        />
+      )}
+    </div>
```

Comment:
I can't spot it from looking at this code, but try entering 0 in either the min or max field and see what happens. Definitely doesn't seem intended.